pkg/coreapi/gql.query.graphql

```diff
@@ -6,17 +6,11 @@ type Query {
   # Get an individual event
   event(query: EventQuery!): Event
 
-  # Get all events sent
-  events(query: EventsQuery!): [Event!]
```

Comment:
TS SDK uses this for integration tests atm 😔

I've added INN-2842 to move to the REST API, though the tests still need detailed state by fetching the timeline 🤔